src/pages/ShopList/ShopListHeader.scss

```
@@ -0,0 +1,42 @@
+.headerContainer {
+  height: 100%;
+  padding-top: 61px;
+
+  .headLine {
+    height: 240px;
+    background-color: #f7f7f7;
+    border-bottom: 1px solid #cecece;
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    flex-direction: column;
+
+    .innerWrapper {
+      height: 100%;
+      padding: 48px 30px 10px 30px;
```

```suggestion
      padding: 48px 30px 10px;
```
위와 같이 쓸 수 있습니다.

https://developer.mozilla.org/ko/docs/Web/CSS/padding

---

중복되는 값 수정했습니다!